src/main/webapp/WEB-INF/tags/student/profile/uploadPhotoModal.tag

```diff
@@ -54,8 +53,23 @@
                             </c:when>
                             <c:otherwise>
                                 <div class="profile-pic-edit">
-                                    <img id="editableProfilePicture" src="${modal.pictureUrl}">
-                                    <br><br>
+                                    <div class="profile-pic-edit-container" style="width: 60%; margin: 0 auto; margin-bottom: 10px;">
```

Comment:
please move the inline styles to a css file by adding an appropriate class or modifying rules in the profile-pic-edit-container class